nni/algorithms/compression/v2/pytorch/pruning/tools/sparsity_allocator.py

```diff
@@ -75,15 +75,15 @@ def generate_sparsity(self, metrics: Dict[str, Tensor]) -> Dict[str, Dict[str, T
             mask = torch.zeros(metric.shape).type_as(metric)
             loop_iters = [range(int(i / j)) for i, j in zip(metric.shape, balance_gran)]
             for iter_params in itertools.product(*loop_iters):
-                index_str_list = [f"{iter_param * gran}:{(iter_param+1) * gran}"\
-                     for iter_param, gran in zip(iter_params, balance_gran)]
+                index_str_list = [f"{iter_param * gran}:{(iter_param+1) * gran}"
+                                  for iter_param, gran in zip(iter_params, balance_gran)]
                 index_str = ",".join(index_str_list)
                 sub_metric_str = "metric[{}]".format(index_str)
-                sub_mask_str =  "mask[{}] = mask_bank".format(index_str)
+                sub_mask_str = "mask[{}] = mask_bank".format(index_str)
```

Comment:
Why `block_sparse_size` is not used in the following `generate_sparsity` function?

Reply:
it is used in `self._compress_mask` and `self._expand_mask`